install_config/install/deploy_router.adoc

```diff
@@ -269,9 +279,89 @@ $ iptables -A INPUT -p tcp --dport 10443 -j ACCEPT
 ====
 
 
+[[working-with-multiple-routers]]
+=== Working With Multiple Routers
+
+Multiple routers can be used to distribute routing load in the cluster. Because
```

Comment:
Hm... not sure if that is entirely the reason.  You can run multiple load-balanced routers that don't use sharding.

Security is certainly part of the picture.  We need to expand this motivation more... might be worth getting @ramr to weigh in if you can't get the information from the trello card he did the work for (you will need to search the archived cards for sharding).